pipeline/db/migrations/20210411132055_create_benchmarks_new_table.up.sql

```diff
@@ -0,0 +1,16 @@
+
+
+CREATE TABLE benchmarks_new (
+  started_at TIMESTAMP WITHOUT TIME ZONE DEFAULT CURRENT_TIMESTAMP,
+  repo_id CHARACTER VARYING(256) NOT NULL,
+  commit CHARACTER VARYING(50) NOT NULL,
+  branch CHARACTER VARYING(256) DEFAULT NULL,
+  pull_number INTEGER DEFAULT NULL,
+  build_job_id CHARACTER VARYING(256) NOT NULL,
+  run_job_id CHARACTER VARYING(256) DEFAULT NULL,
+  status json DEFAULT NULL,
```

Comment:
Why did you change the status to be json?

Reply:
This way we can store errors in a more uniform way and present something useful in the UI. See: https://github.com/ocurrent/current-bench/pull/120/files#diff-9769a3f5504af3cd735e08f078b94f827bdd51eacac2e9d80ac9761bc485171fR249